cmd/api/src/database/db.go

```diff
@@ -52,26 +54,25 @@ type Database interface {
 	appcfg.FeatureFlagService
 
 	Close()
-	CreateIngestTask(ingestTask model.IngestTask) (model.IngestTask, error)
-	GetAllIngestTasks() (model.IngestTasks, error)
-	DeleteIngestTask(ingestTask model.IngestTask) error
-	GetIngestTasksForJob(jobID int64) (model.IngestTasks, error)
-	GetUnfinishedIngestIDs() ([]int64, error)
+
+	// Ingest
+	ingest.IngestData
+	GetAllIngestTasks(ctx context.Context) (model.IngestTasks, error)
+	DeleteIngestTask(ctx context.Context, ingestTask model.IngestTask) error
+	GetIngestTasksForJob(ctx context.Context, jobID int64) (model.IngestTasks, error)
 
 	// Asset Groups
+	agi.AgiData
```

Comment:
Missed cleaning this up in the asset-groups ctx PR but since these methods are defined elsewhere, having them on this interface is unnecessary.

Reply:
I'm confused here, too, sorry. Your comment makes it sound like this line should be deleted, but this is showing as an addition on the diff.

Reply:
The 3 methods that are duplicated beneath it are encompassed within the agi.AgiData:
`GetAssetGroup`
`GetAllAssetGroups`
`CreateAssetGroupCollection`